content/billing/managing-your-license-for-github-enterprise/downloading-your-license-for-github-enterprise.md

```diff
@@ -18,7 +18,13 @@ After you purchase or upgrade a license for {% data variables.product.prodname_e
 
 {% data reusables.enterprise-licensing.contact-sales-for-renewals-or-seats %}
 
-## Downloading your license from {% data variables.product.prodname_dotcom_the_website %}
+## Downloading your license if you don't have an enterprise account on {% data variables.product.prodname_dotcom_the_website %}
+
+If you do not have an enterprise account on {% data variables.product.prodname_dotcom_the_website %}, or if you're not sure, you may be able to download your {% data variables.product.prodname_ghe_server %} license from the [{% data variables.product.prodname_enterprise %} website](https://enterprise.github.com/download).
+
+If you have any questions about downloading your license, contact {% data variables.contact.contact_enterprise_sales %}.
+
+## Downloading your license from {% data variables.product.prodname_dotcom_the_website %} if you have an enterprise account
```

Comment:
I think both the previous section's title and the first sentence in this section make the requirement clear, so I'd advise keeping this heading a bit shorter.

```suggestion
## Downloading your license from {% data variables.product.prodname_dotcom_the_website %}
```